<launch>
    <node pkg="netft_rdt_driver" name="ft10" type="netft_node" args="192.168.0.126" respawn="true">
        <remap from='netft_data' to='/force_torque_ft10' />
    </node>

    <node name="static_tf_ft10_broadcaster" pkg="tf"
        type="static_transform_publisher"
        args="0 0 0 0 0 0 1 world force_torque_ft10 50"
        respawn="false" output="screen"/>
</launch>

